ui_framework/components/card/_card_group.scss

```diff
@@ -1,23 +1,18 @@
 /**
- * 1. Use this container to offset the spacing between wrapped cards.
- */
-.kuiCardGroupContainer {
-  margin: -$cardSpacing; /* 1 */
-}
-
-/**
  * 1. Wrap cards when necessary.
+ * 2. Offset the spacing between wrapped cards.
  */
 .kuiCardGroup {
   display: flex;
   flex-wrap: wrap; /* 1 */
+  margin: -$cardSpacing; /* 2 */
 
   /**
    * 1. Use the defined width of the card to determine when to wrap.
    * 2. Use an even margin all around the card so that the spacing is still even when wrapped.
    */
   .kuiCard {
```

Comment:
@cjcenizal, thanks for the historic references. I tend to agree with you and @PopradiArpad that this looks like a good exemption from the rule. We should definitely document that somewhere. I will create a PR to amend the css style guide.